src/main/java/UserManager.java

```diff
@@ -84,6 +86,21 @@ public static void changeUserName(User user, String newName) throws Exception {
         user.setUserName(newName);
 //        UserParser.updateUserInfo();
     }
+
+    public static void changeInfo(User user, Object info, int command){
+        switch(command){
```

Comment:
I assume this switch operation is for the different functionalities that the user can choose. If that's the case, shouldn't we add case "1" here as well in case the user chose the BMI option?